### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Soeparmadi** [symfony/symfony#45501] [Q&A]
Error when completed address
![Screenshot_2022-02-21-19-54-20-13_439a3fec0400f8974d35eed09a31f914](https://user-images.githubusercontent.com/91101378/154992789-21e51d6a-a221-4123-8141-febbcd4dc81d.jpg)
What happen?

**Comment by chyshyya** [comment]
Something you try to access country field is null or undefined at all

**Reply by osavchenko** [reply]
I'd like to say that _user_ is null and there is no possibility to get `country` of non-existing user (null)
